### ISSUE TITLE
Whether the extension is unfriendly for workspac？

### ISSUE THREAD

**Issue by successismyonlyway** [redhat-developer/vscode-java#3382]
Problems occurred frequently.

**Comment by fbricon** [comment]
No, it's workspace friendly. You need to provide actual information about your issues if you expect some help.

**Comment by successismyonlyway** [comment]
> No, it's workspace friendly. You need to provide actual information about your issues if you expect some help.

A lot of issues.

**Comment by successismyonlyway** [comment]
such as not found java project.

**Comment by rgrunber** [comment]
Do you have a sample project you could share that shows this behaviour ?

**Comment by successismyonlyway** [comment]
> Do you have a sample project you could share that shows this behaviour ?

Currently, i am not using workspaces and I have switch to the settings profiles.

**Comment by successismyonlyway** [comment]
It seems running ok at all.
